### PR TITLE
Updating Script repo with latest ApiHub SDK

### PR DESCRIPTION
**Pull request by safihamid**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@safihamid__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (hamids). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: sample/ApiHubTrigger-Batch/function.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
 {
   "bindings": [
     {
-      "type": "apiHubTrigger",
+      "type": "apiHubFileTrigger",
```
Comment (mathewc) [inline sample/ApiHubTrigger-Batch/function.json]: Recommend renaming your sample dirs as well, e.g. "ApiHubFileTrigger".

File: src/WebJobs.Script/Description/Binding/ApiHubBindingMetadata.cs [hunk 2]
```diff
@@ -14,6 +14,8 @@ public class ApiHubBindingMetadata : BindingMetadata
         [JsonIgnore]
         public string Key { get; set; }
 
+        public int PollIntervalInSeconds { get; set; }
```
Comment (mathewc) [inline src/WebJobs.Script/Description/Binding/ApiHubBindingMetadata.cs]: You don't apply [JsonIgnore] on this one, but you do on the one above. I'm wondering why the above ignore attribute even exists - I don't see any of the other metadata classes using this.

File: test/WebJobs.Script.Tests/TestScripts/CSharp/ApiHubTrigger/function.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
```
Comment (mathewc) [inline test/WebJobs.Script.Tests/TestScripts/CSharp/ApiHubTrigger/function.json]: Same folder renames suggested here as well "ApiHubFileTrigger"

**Comment by mathewc** [comment]
Few nit comments - rest looks ok

File: sample/ApiHubTrigger-Batch/function.json (hunk 1 above)
Comment (safihamid) [inline sample/ApiHubTrigger-Batch/function.json]: this is done. but still shows as open item.

File: test/WebJobs.Script.Tests/TestScripts/CSharp/ApiHubTrigger/function.json (hunk 3 above)
Comment (safihamid) [inline test/WebJobs.Script.Tests/TestScripts/CSharp/ApiHubTrigger/function.json]: same as this one. it's done.

**Comment by davidebbo** [comment]
History is looking messed up, with head having two parents with same commit string:

image

Can you fix? Thanks!

**Comment by safihamid** [comment]
Can you check again please?

File: test/WebJobs.Script.Tests/FileTraceWriterTests.cs [hunk 4]
```diff
@@ -75,6 +75,8 @@ public FileTraceWriterTests()
             File.SetLastWriteTime(logFiles[1].FullName, DateTime.Now.Subtract(TimeSpan.FromDays(1)));
             File.SetLastWriteTime(logFiles[0].FullName, DateTime.Now.Subtract(TimeSpan.FromDays(2)));
 
+            await Task.Delay(2000);
```
Comment (mathewc) [inline test/WebJobs.Script.Tests/FileTraceWriterTests.cs]: This isn't added by your commit, but by Fabio's latest commit - not sure why this is showing up as an addition by you. Maybe you just need to rebase.

**Comment by mathewc** [comment]
:ship:

**Pull request merged**